### ISSUE TITLE
NullReferenceException when deleting a row with a comment in it (regression)

### ISSUE THREAD

**Issue by coxymla** [JanKallman/EPPlus#70]
We recently upgraded our EPPlus to latest from 4.0.4, and the library seems to have introduced a regression.
It happens when deleting a row that has a comment in it.

```
System.NullReferenceException : Object reference not set to an instance of an object.
   at OfficeOpenXml.ExcelCommentCollection.Delete(Int32 fromRow, Int32 fromCol, Int32 rows, Int32 columns)
   at OfficeOpenXml.ExcelWorksheet.DeleteRow(Int32 rowFrom, Int32 rows)
```
Minimal repro:
```
var documentPath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, @"..\..\..\workbook with comment.xlsx");
var outputPath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, @"..\..\..\WorkbookWithCommentOutput.xlsx");
var fileInfo = new FileInfo(documentPath);
Assert.IsTrue(fileInfo.Exists);
using (var workbook = new ExcelPackage(fileInfo))
{
    var ws = workbook.Workbook.Worksheets.First();
    ws.DeleteRow(3); // NRE thrown here
    workbook.SaveAs(new FileInfo(outputPath));
}
```
[workbook with comment.xlsx](https://github.com/JanKallman/EPPlus/files/1500813/workbook.with.comment.xlsx)

**Comment by JanKallman** [comment]
Fixed in 4.5.1

**Comment by Prinsn** [comment]
I'm observing this in 4.5.3.2
